backend/src/resolvers/submissionResolvers.js

```diff
@@ -19,10 +19,13 @@ export default {
       await Submission.updateOne({ _id: mongoose.Types.ObjectId(id) }, { $set: { grade } });
       return grade;
     },
-    // peerGradeSubmission: async (_, { peerGradeInput }) => {
-    //   creates a new peer grade if the grader hasn't graded the assignment yet
-    //   otherwise updates the student's grade
-    //   return peerGradeInput; // TODO
-    //}
+
+    updateSubmission: async (_, { id, submission }) => {
+      return await Submission.findByIdAndUpdate(id, submission);
+    },
+
+    deleteSubmission: async (_, { id }) => {
+      return await Submission.findByIdAndDelete(id);
+    },
```

Comment:
We ended up removing the "Submission" model. Not your fault, I should've let you know. Ill leave the PR open for now but I think we ended up doing duplicate work. I need to let you know next time, this is 100% on me.